cmd/mothership/mothership/start/helpers.go

```diff
@@ -19,40 +18,3 @@ func components(cfg model.ClusterConfigurationEntity) []keb.Component {
 	}
 	return components
 }
-
-func clusterMetadata(runtimeID string, lastState *cluster.State) *keb.Cluster {
```

Comment:
We should not drop the whole `cluster` part of the response.. but just the kubeconfig.
See https://github.com/kyma-incubator/reconciler/issues/433, where @tobiscr expects non empty `components` list as part of that object which is here being removed

Reply:
The components config will be implemented in this PR in another endpoint:
https://github.com/kyma-incubator/reconciler/pull/437